### ISSUE TITLE
Determine the number type from a JsonNode for following datatype

### ISSUE THREAD

**Issue by shivarm** [Hyperfoil/Horreum#969]
look at- https://github.com/Hyperfoil/Horreum/blob/3d5bc4d9a6e923a41158f08b19dc8a7fa8b2e174/horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/data/FingerprintValue.java#L42

Determine: INT, FLOAT, DOUBLE

**Comment by shivam-sharma7** [comment]
@jesperpedersen WDYT?

**Comment by shivam-sharma7** [comment]
Agree!, number is int type then we can do- https://github.com/FasterXML/jackson-databind/blob/31dc75f082e225863d4e2f033f5df9a9daf062a9/src/main/java/com/fasterxml/jackson/databind/node/NumericNode.java#L48

for Numeric value

```java
 v.value = node.asInt();
```

instead 
```java
 v.value = node.asDouble();
```

what are we doing currently, check here-https://github.com/Hyperfoil/Horreum/blob/3d5bc4d9a6e923a41158f08b19dc8a7fa8b2e174/horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/data/FingerprintValue.java#L44
